domains/freetux.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "herotux",
+        "email": "thefreetux@gmail.com"
+    },
+    "record": {
+      "A": ["116.202.215.237"]
```

Comment:
I cannot reach your website, "116.202.215.237 refused to connect.", can you show a preview of it?